lgsm/functions/install_server_files.sh

```diff
@@ -85,6 +85,14 @@ fn_install_server_files() {
 		chmodx="nochmodx" run="norun"
 		force="noforce"
 		md5="7c664538999252eeaf2b6d9949416480"
+    elif [ "${shortname}" == "tf2c" ]; then
+		remote_fileurl="https://chapo.services/tf2c/tf2classic-2.0.3.zip" 
```

Comment:
Using zip file it's fine. ZIP is supported by LGSM so in this case, tar.gz is not needed.
But in my opinion, more official source is https://tf2classic.org/tf2c/tf2classic-2.0.3.zip

But both of this source have some problem. They use an uncommon compression algorithm that is not supported out of the box by most of distro. They also provide .tar.zst files https://tf2classic.org/tf2c/ but .zst (https://github.com/facebook/zstd) is also not supported.

You also should add `tf2c` to this if condition to allow lgsm to download this zip
https://github.com/GameServerManagers/LinuxGSM/blob/141b6d9772e83be8813813270b2fd27c75f5d69b/lgsm/functions/install_server_files.sh#L246